### ISSUE TITLE
Failed to activate the node-on-fire package

### ISSUE THREAD

**Issue by kogutu** [martijndeh/node-on-fire-atom-package#1]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 0.209.0
**System**: Microsoft Windows 8.1
**Thrown From**: [node-on-fire](https://github.com/martijndeh/node-on-fire-atom-package) package, v0.0.2
### Stack Trace

Failed to activate the node-on-fire package

```
At Cannot find module 'atom:\config\package.json'

Error: Cannot find module 'atom:\config\package.json'
    at Module._resolveFilename (module.js:328:15)
    at Function.Module._resolveFilename (C:\Users\DirectSEO\AppData\Local\atom\app-0.209.0\resources\app.asar\src\module-cache.js:383:52)
    at Function.Module._load (module.js:270:25)
    at Module.require (module.js:357:17)
    at require (module.js:376:17)
    at Firestarter.isProject (C:\Users\DirectSEO\.atom\packages\node-on-fire\lib\firestarter.js:185:21)
    at Firestarter.activate (C:\Users\DirectSEO\.atom\packages\node-on-fire\lib\firestarter.js:193:10)
    at Package.module.exports.Package.activateNow (C:\Users\DirectSEO\AppData\Local\atom\app-0.209.0\resources\app.asar\src\package.js:240:19)
    at C:\Users\DirectSEO\AppData\Local\atom\app-0.209.0\resources\app.asar\src\package.js:221:30
    at Package.module.exports.Package.measure (C:\Users\DirectSEO\AppData\Local\atom\app-0.209.0\resources\app.asar\src\package.js:165:15)
```
### Commands

```
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-light-ui",
      "atom-light-syntax"
    ],
    "disabledPackages": [
      "linter-eslint",
      "language-js-specs"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
atom-jshint, v2.0.0
comment-down, v0.1.1
emmet, v2.3.10
javascript-snippets, v1.0.0
jshint, v1.3.7
jslint, v1.4.0
node-on-fire, v0.0.2
remote-edit, v1.8.2
Remote-FTP, v0.7.1
scope-inspector, v1.0.1

# Dev
No dev packages
```

**Comment by abougouffa** [comment]
[Enter steps to reproduce below:]
1. Open Atom
2. Atom automaticly load the last working directory (I have the [save-session](https://github.com/mpeterson2/save-session) package installed)

**Atom Version**: 1.0.0
**System**: linux 4.0.4-2-ARCH
**Thrown From**: [node-on-fire](https://github.com/martijndeh/node-on-fire-atom-package) package, v0.0.2

### Stack Trace

Failed to activate the node-on-fire package

```
At Cannot find module '/home/hacko/PFE/pfeman/package.json'

Error: Cannot find module '/home/hacko/PFE/pfeman/package.json'
    at Module._resolveFilename (module.js:328:15)
    at Function.Module._resolveFilename (/usr/share/atom/resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:270:25)
    at Module.require (module.js:357:17)
    at require (module.js:376:17)
    at Firestarter.isProject (/home/hacko/.atom/packages/node-on-fire/lib/firestarter.js:185:21)
    at Firestarter.activate (/home/hacko/.atom/packages/node-on-fire/lib/firestarter.js:193:10)
    at Package.module.exports.Package.activateNow (/usr/share/atom/resources/app.asar/src/package.js:240:19)
    at /usr/share/atom/resources/app.asar/src/package.js:221:30
    at Package.module.exports.Package.measure (/usr/share/atom/resources/app.asar/src/package.js:165:15)
    at Package.module.exports.Package.activate (/usr/share/atom/resources/app.asar/src/package.js:213:14)
    at PackageManager.module.exports.PackageManager.activatePackage (/usr/share/atom/resources/app.asar/src/package-manager.js:452:21)
    at /usr/share/atom/resources/app.asar/src/package-manager.js:436:29
    at Config.module.exports.Config.transact (/usr/share/atom/resources/app.asar/src/config.js:311:16)
    at PackageManager.module.exports.PackageManager.activatePackages (/usr/share/atom/resources/app.asar/src/package-manager.js:431:19)
    at PackageManager.module.exports.PackageManager.activate (/usr/share/atom/resources/app.asar/src/package-manager.js:412:46)
```

### Commands

```
```

### Config

``` json
{
  "core": {
    "disabledPackages": [
      "tool-bar-main",
      "tool-bar",
      "flex-tool-bar",
      "node-debugger",
      "node-resolver",
      "node-resolver",
      "bootstrap-3-snippetset"
    ]
  }
}
```

### Installed Packages

``` coffee
# User
atom-ternjs, v0.6.8
atom-zeal, v0.1.3
autocomplete-modules, v1.0.0
autocomplete-paths, v1.0.2
autocomplete-php, v0.3.6
autocomplete-python, v0.5.1
autocomplete-sass, v0.1.0
browser-plus, v0.0.23
data-atom, v0.6.1
emmet, v2.3.12
file-type-icons, v0.7.0
javascript-snippets, v1.0.0
make-coffee, v0.1.1
minimap, v4.10.2
node-on-fire, v0.0.2
save-session, v0.15.6
tasks, v2.1.0
unitjs, v1.0.0

# Dev
No dev packages
```

**Comment by martijndeh** [comment]
Thanks for reporting this issue. This has been resolved in `v0.1.0`. Can you please confirm?
